package.json

```diff
@@ -2,7 +2,7 @@
   "name": "kontena-lens",
   "productName": "Lens",
   "description": "Lens - The Kubernetes IDE",
-  "version": "5.0.0-alpha.1",
```

Comment:
Why is this changed in this PR?

Reply:
Migration is not executed without bumping version. Is there better way to achieve this?

Reply:
ah I see, makes sense. I was just confused because this wasn't a release PR